.github/workflows/testing.yml

```diff
@@ -0,0 +1,20 @@
+name: Testing
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.14
+
+      - name: Run tests
+        run: go test -v ./...
```

Comment:
Should we also add an action which runs `gofmt` or `go vet`?

Reply:
Maybe, but "This tries to be 1:1 with no behavior differences except where it is run."

First I'd like to bump the go version.